docs/updates/migrate-v3.md

```diff
@@ -67,53 +67,52 @@ await graph.build({
 The following tasks have been removed:
 
 - createDebugFiles
+- generateManifestBundle
 - uglify
 
 The following processors have been removed:
 
 - debugFileCreator
+- manifestBundler
 - resourceCopier
 - uglifier
 
-As a replacement, the new 'minify' task and 'minifier' processor can be
-used.
+**Task Migration**
 
-!!! note
-    The minify task is executed earlier, before the bundling
-    process takes place. Any existing 'beforeTask' or 'afterTask' configuration of
-    custom tasks might need to be adapted to cater for this change.
+| UI5 Tooling v2              | UI5 Tooling v3              | Note |
+| --------------------------- | --------------------------- | ------------------------- |
+| createDebugFiles<br/>uglify | minify                      | The minify task is executed earlier, before the bundling process takes place. Any existing 'beforeTask' or 'afterTask' configuration of custom tasks might need to be adapted to cater for this change. |
+| generateVersionInfo         | generateVersionInfo         | The task is no longer executed for application projects by default. It can be re-enable by using the `--include-task` parameter. |
+| generateManifestBundle      | *None*                      | This task was only needed for the HTML5 repository in Cloud Foundry. In the meantime the HTML5 repository implemented an own mechanism, so the task is no longer needed |
```

Comment:
| generateManifestBundle      | *None*                      | This task was only needed for the HTML5 repository in Cloud Foundry. Meanwhile, the HTML5 repository implemented its own mechanism, so the task is no longer needed |

Reply:
https://github.com/SAP/ui5-tooling/pull/694